src/helm/benchmark/scenarios/cleva_scenario.py

```diff
@@ -689,18 +699,6 @@ class CLEVAParaphraseGenerationScenario(CLEVAScenario):
     description = "Paraphrase generation task in CLEVA benchmark"
     tags = ["paraphrase_generation"]
 
-    def process_instance(self, row: Dict[str, Any], split: str) -> Instance:
```

Comment:
Do not remove this: It is necessary for the paraphrase generation scenario to overwrite this method as we use HELM's prompting strategy here.